src/main/java/com/acoustic/SpringPolandSalaryCalculator/calculatorservice/DisabilityZusService.java

```diff
@@ -0,0 +1,32 @@
+package com.acoustic.SpringPolandSalaryCalculator.calculatorservice;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+import org.springframework.stereotype.Component;
+
+import com.acoustic.SpringPolandSalaryCalculator.rates.RatesConfigurationProperties;
+import lombok.RequiredArgsConstructor;
+
+
+@Component
+@RequiredArgsConstructor
+public class DisabilityZusService implements SalaryCalculatorService {
+
+    private final RatesConfigurationProperties rate;
+
+    @Override
+    public BigDecimal apply(BigDecimal grossMonthlySalary) {
+        return grossMonthlySalary.multiply(this.rate.getDisabilityZusRate()).setScale(2, RoundingMode.HALF_EVEN);
+    }
+
+    @Override
+    public int getCalculationOrder() {
+        return 6;
```

Comment:
Can calculation be done parallel? AnnualGrossService.java has the same order.

Reply:
The calculation cannot be done in parallel, I have updated from 6 to 7 - Done